templates/branding/openqa.suse.de/sponsorbox.html.ep

```diff
@@ -0,0 +1,5 @@
+<div class="text-right" id="sponsorbox">
```

Comment:
I am pretty sure no but I thought it would be more correct as SUSE is not a *sponsor* of openqa.suse.de, right?